packages/website/docs/getting-started.md

```diff
@@ -3,75 +3,252 @@ id: getting-started
-
-- These docs provide a few ways to learn how to use Autocomplete:
-  - Read about **Core Concepts**—here you can learn more about underlying principles, like **Sources** and **State**.
-  - Follow our **Guides** to understand how to build common UX patterns.
-  - Refer to **API reference**.
-  - [Maybe v2] Play in the **Playground** where you can select components of your autocomplete and we provide you with the code.
-- Keep reading for a simple sample implementation
-- Installation
-  - JS
-  - Core
-- A basic example
-  - Commented code snippet on a basic example (only using query-suggestions plug-in) with resulting UI and links to various **Core Concepts.**
+Get started with Autocomplete by building an Algolia search experience.
 
-:::
-
-This page is an overview of the Autocomplete documentation and related resources.
-
-Autocomplete is a JavaScript library for **building autocomplete search experiences**.
-
-## Features
+This documentation offers a few ways to learn about the Autocomplete library:
+  - Read the [**Core Concepts**](basic-options) to learn more about underlying principles, like [**Sources**](sources) and [**State**](state).
+  - Follow the [**Guides**](using-query-suggestions-plugin) to understand how to build common UX patterns.
+  - Refer to [**API reference**](api) for a comprehensive list of parameters and options.
+  - Try out the [**Playground**](https://codesandbox.io/s/github/algolia/autocomplete.js/tree/next/examples/js?file=/app.ts) where you can fork a basic implementation and play around.
 
-- Displays suggestions as you type
-- Provides autocompletion
-- Supports custom templates for UI flexibility
-- Works well with RTL languages
-- Triggers custom hooks to plug your logic
-- Plugs easily to Algolia's realtime search engine
-
-## What is Autocomplete
+Keep reading to see how to install Autocomplete and build a basic implementation with Algolia.
 
 ## Installation
 
-Autocomplete is available on the [npm](https://www.npmjs.com/) registry.
+The recommended way to get started is with the [`autocomplete-js`](autocomplete-js) package. It includes everything you need to render a JavaScript autocomplete experience.
+
+Otherwise, you can install the [`autocomplete-core`](createAutocomplete) package if you want to [build a renderer](creating-a-renderer) from scratch.
 
-### JavaScript
+All Autocomplete packages are available on the [npm](https://www.npmjs.com/) registry.
 
 ```bash
 yarn add @algolia/autocomplete-js@alpha
 # or
 npm install @algolia/autocomplete-js@alpha
 ```
 
-If you do not wish to use a package manager, you can use standalone endpoints:
+If you don't want to use a package manager, you can use a standalone endpoint:
 
 ```html
-<!-- jsDelivr -->
 <script src="https://cdn.jsdelivr.net/npm/@algolia/autocomplete-js@alpha"></script>
+```
+
+We recommend using jsDeliver but [`autocomplete-js`](autocomplete-js) is also available through [unpkg](https://unpkg.com/@algolia/autocomplete-js@1.0.0-alpha.39).
```

Comment:
So that we don't deal with versioning for now:

```suggestion
We recommend using jsDeliver but [`autocomplete-js`](autocomplete-js) is also available through [unpkg](https://unpkg.com/@algolia/autocomplete-js@alpha).
```